### ISSUE TITLE
Error using expression for input in onResume

### ISSUE THREAD

**Issue by stumpf84** [kestra-io/kestra#5584]
Using an expression to fill SELECT and MULTISELECT inputs values causes an error.
In this task an array of hosts is retreived from a ansible host.yml file.
These hosts are used as values for an input.

```
id: input_sample
namespace: sample

tasks:

  - id: get_hosts_subflow
    type: io.kestra.plugin.core.flow.Subflow
    namespace: sample
    flowId: get_all_hosts

  - id: dbg
    type: io.kestra.plugin.core.log.Log
    message: "{{ outputs.get_hosts_subflow.outputs.hosts }}"  

  - id: select_host
    type: io.kestra.plugin.core.flow.Pause
    onResume:
      - id: host
        type: SELECT
        expression: "{{ outputs.get_hosts_subflow.outputs.hosts }}"
        required: true

  - id: selected_host
    type: io.kestra.plugin.core.log.Log
    message: "{{ outputs.select_host.onResume.host }}"
```
Logging the hosts in the dbg task shows the correct values.
![image](https://github.com/user-attachments/assets/7db5ce22-54b1-480f-9b0a-6ab27d775990)

Resuming the flow causes an error:
 Illegal state
Illegal state: refCnt: 0, decrement: 1
Resulting in an empty input.
![image](https://github.com/user-attachments/assets/fc50b59d-5a4e-4c1f-9a3a-af4ce6d60a2a)

The flow worked as expected on Friday but since today, after updating from v0.19.3 to v0.19.4, it is not.
Using the v0.19.3 image results in the same error.
I'm not sure if I did something different on Friday when creating the flow and it worked or if this is a bug.

### Environment

- Kestra Version: v0.19.4
- Operating System (OS/Docker/Kubernetes): Docker
- Java Version (if you don't run kestra in Docker):

**Comment by Smiloutcha** [comment]
Hello, 
I want to contribute , can you please give me the ticket , the scénario to reproduce , also the steps to debug on kestra 
kind regards,

**Comment by MilosPaunovic** [comment]
This might need internal handling @Smiloutcha, but feel free to take your pick from the curated [list of good first issues](https://github.com/orgs/kestra-io/projects/15/views/1?filterQuery=no%3Aassignee+label%3A%22good+first+issue%22 )and take your pick.

**Comment by loicmathieu** [comment]
@stumpf84 this has already been fixed by https://github.com/kestra-io/kestra/pull/5498
It will be included in 0.19.5 that should be published today.
